peachjam/migrations/0075_backfill_authors.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.18 on 2023-05-16 11:30
+
+from django.db import migrations
+
+
+def backfill_authors(apps, schema_editor):
+    CoreDocument = apps.get_model("peachjam", "CoreDocument")
+    for doc in CoreDocument.objects.all().iterator(chunk_size=100):
```

Comment:
```suggestion
    for doc in CoreDocument.objects.only('author').all().iterator(chunk_size=100):
```

otherwise the entire content is loaded which is slow.

Reply:
You will need to explicitly do this for `GenericDocument` and `LegalInstrument`, since 'author' is not a field on `CoreDocument`